databases/week2/1hyf_lesson2_class_exercise.sql

```diff
@@ -0,0 +1,38 @@
+-- Get all the tasks assigned to Pavel;
+SELECT
+  task.title,
+  user.name
+FROM
+  (
```

Comment:
the protestants are not necessary, if you think it can help the readability keep it, otherwise you may remove it